web/src/main/kotlin/com/example/ajaxproject/repository/RedisPrivateChatRoomRepository.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.ajaxproject.repository
+
+import com.example.ajaxproject.model.PrivateChatRoom
+import reactor.core.publisher.Mono
+
+interface RedisPrivateChatRoomRepository {
```

Comment:
Don't specify infrastructure details inside an interface name. 

`Redis` is infrastructure detail.

`RedisPrivateChatRoomRepository` -> `CacheableRoomRepository` as example.